### PR TITLE
feat: create single backcover page content

### LINKED ISSUE #11: Generate single backcover

### PR DESCRIPTION
**Pull request by pyphilia**
closes #11

### CODE REVIEW

File: templates/epub3/content.opf.ejs [hunk 1]
```diff
@@ -54,13 +54,20 @@
...
+        <% } %>
+
         <% fonts.forEach(function(font, index){%>
         <item id="font_<%= index%>" href="fonts/<%= font %>" media-type="application/x-font-ttf" />
         <%})%>
     </manifest>
 
     <spine toc="ncx">
         <itemref idref="toc" />
+        <% if(backCover){ %>
```
Comment (juancarlosfarah) [inline templates/epub3/content.opf.ejs]: Space after `if`

File: templates/epub3/content.opf.ejs [hunk 2]
```diff
@@ -54,13 +54,20 @@
             <% }) %>
         <% }) %>
 
+        <% if(backCover){ %>
```
Comment (juancarlosfarah) [inline templates/epub3/content.opf.ejs]: Space after `if`

File: lib/index.js [hunk 3]
```diff
@@ -277,6 +283,11 @@
           });
       });
 
+      
+      if(this.options.backCover) {
```
Comment (juancarlosfarah) [inline lib/index.js]: Space after `if`

File: lib/index.js [hunk 4]
```diff
@@ -101,6 +101,11 @@
         return section;
       });
 
+      if(this.options.backCover) {
```
Comment (juancarlosfarah) [inline lib/index.js]: Space after `if`

**Review by juancarlosfarah — CHANGES_REQUESTED** [review]
Some very minor changes. Also, have you tested that this doesn't affect the single-space exports?

**Comment by pyphilia** [comment]
Yes I always test both.

**Review by juancarlosfarah — APPROVED** [review]

**Pull request merged**